src/west/commands/project.py

```diff
@@ -120,7 +120,8 @@ def do_add_parser(self, parser_adder):
             - url: full remote URL as specified by the manifest
             - path: the relative path to the project from the top level,
               as specified in the manifest where applicable
-            - abspath: absolute and normalized path to the project
+            - abspath: absolute and normalized path to the project in posix
```

Comment:
I think this is likely to mess up other tools.

Can we add a new `{posixpath}` key instead?